File: SofiForce.Models/Models/EntityModels/PromotionMixGroupModel.cs
using System;

namespace Models
{
    public class PromotionMixGroupModel
    {
        public int GroupId { get; set; }
        public int? GroupNo { get; set; }

        public int? PromotionId { get; set; }
        public decimal? Slice { get; set; }
        public bool? IsActive { get; set; }

        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromotionMixGroupModel.ts
export class PromotionMixGroupModel {
    groupId: number;
    groupNo: number;

    promotionId: number | null;
    slice: number | null;
    isActive: boolean | null;

    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}